src/connect.js

```diff
@@ -20,7 +20,11 @@
  * @return {Function}
  */
 function connect(params) {
-  // write code here
+  const editedParams = { ...params };
+  return (extraParams) => {
```

Comment:
you can write simply 
```suggestion
  return (extraParams) => { ...params, ... extraParams };
```